source/extensions/filters/http/ext_proc/ext_proc.cc

```diff
@@ -129,6 +129,15 @@ FilterConfigPerRoute::initGrpcService(const ExtProcPerRoute& config) {
   }
   return absl::nullopt;
 }
+namespace {
+std::vector<envoy::config::core::v3::HeaderValue> initMetadata(const ExtProcPerRoute& config) {
```

Comment:
No need to have this function. I think you can do the initialization in constructor init list like one line below: 
```
metadata_(config.overrides().metadata().begin(), config.overrides().metadata().end()
```

Reply:
done